database/oracle/oracle_test.go

```diff
@@ -0,0 +1,270 @@
+package oracle
+
+import (
+	"bytes"
+	"context"
+	"database/sql"
+	sqldriver "database/sql/driver"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/dhui/dktest"
+	"github.com/docker/docker/api/types/mount"
+	"github.com/docker/go-connections/nat"
+	"github.com/golang-migrate/migrate/v4"
+	dt "github.com/golang-migrate/migrate/v4/database/testing"
+	"github.com/golang-migrate/migrate/v4/dktesting"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	defaultPort = 1521
+	userdba     = "orcl"
+	userdbaPass = "orcl"
+	defaultPass = "orcl"
+)
+
+var (
+	specs = []dktesting.ContainerSpec{
+		{
+			ImageName: "gvenzl/oracle-free:23.5-slim", Options: oracleOptions(),
```

Comment:
I'm not familiar with OracleDB, but is it possible to use an official Docker image? Most of the other database tests use official images.